modules/monitoring-common-terms.adoc

```diff
@@ -0,0 +1,122 @@
+
+Cluster Monitoring Operator::
+The Cluster Monitoring Operator (CMO) is a central component of the monitoring stack. It deploys and manages Prometheus instances, the Thanos Querier, the Telemeter Client, and metrics targets and ensures that they are up to date. The CMO is deployed by the Cluster Version Operator (CVO).
+
+Cluster Version Operator::
+
+CoreDNS::
+
+config map::
+A config map provides a way to inject configuration data into pods. You can reference the data stored in a config map in a volume of type `ConfigMap`. Applications running in a pod can use this data.
+
+
+
+Container::
+It is a lightweight and executable image that comprises software and all its dependencies. Containers virtualize the operating system, as a result, you can run containers anywhere from a data center to a public or private cloud to even a developer’s laptop.
+
+
+custom resource (CR)::
+A CR is extension of the Kubernetes API. You can create custom resources.
+
+Elasticsearch::
+
+etcd::
+
+Federation endpoint::
+
+Fluentd::
+
+HAProxy::
+
+Image registry::
+
+Kubelets::
+Runs on nodes and reads the container manifests. Ensures that the defined containers have started and are running.
+
+Kubernetes API server::
+
+Kubernetes controller manager::
+Kubernetes controller manager governs the state of the cluster.
+
+Kubernetes scheduler::
+Kubernetes scheduler allocates pods to nodes.
+
+labels::
+You can use labels, which are key-value pairs, to organise and select subsets of objects, such as a pod.
+
+Metering::
+
+metrics::
+
+metric target::
+
+monitoring dashboards::
+
+monitoring stack::
+
+node::
+A worker machine in the {product-title} cluster. A node is either a virtual machine (VM) or a physical machine.
+
+NVIDIA GPU administration dashboard::
+
+Operator::
+The preferred method of packaging, deploying, and managing a Kubernetes application in an {product-title} cluster. An Operator takes human operational knowledge and encodes it into software that is packaged and shared with customers.
+
+{product-title} API server::
+
+{product-title} Controller Manager::
+
+Operator Lifecycle Manager (OLM)::
+
+Persistent storage::
+Stores the data even after the device is shut down. Kubernetes uses persistent volumes to store the application data
+
+Persistent volume claim (PVC)::
+You can use a PVC to mount a PersistentVolume into a Pod. You can access the storage without knowing the details of the cloud environment.
+
+pod::
+The pod is the smallest logical unit in Kubernetes. A pod contains one or more containers to run in a worker node.
+
+project::
+
+Prometheus::
+Prometheus is the monitoring system on which the OpenShift Container Platform monitoring stack is based. Prometheus is a time-series database and a rule evaluation engine for metrics. Prometheus sends alerts to Alertmanager for processing.
+
+Prometheus adapter::
```

Comment:
see description for `Prometheus adapter`
https://docs.openshift.com/container-platform/4.11/monitoring/monitoring-overview.html#default-monitoring-components_monitoring-overview

Reply:
Addressed

Reply:
The Prometheus Adapter (PA in the preceding diagram) translates Kubernetes node and pod queries for use in Prometheus. The resource metrics that are translated include CPU and memory utilization metrics. The Prometheus Adapter exposes the cluster resource metrics API for horizontal pod autoscaling.

Reply:
Addressed